Minecraft_Quick_Homes/Minecraft_Quick_Homes.ahk

```diff
@@ -345,6 +395,281 @@ Wait_Until_Minecraft_Registers_Slash()
+            ; msgbox % home_sections.1[1] ; array[1] is useless, since config will already be in there. 
+            ; msgbox % home_sections.1[2] ; However, Array[2] and so forth, will highlight all the other ones.
+            
+            ; | Defining the subsection stored in this variable.
+            ; |                                     | Parsing the sections into subsections **To Be Stored**.
+            ; \/                                    \/
+            ; home_sections.2/*[Subsection_ID]*/ := StrSplit(home_sections.1[2], "&")
+            ; home_sections[2]/*[Subsection_ID]*/ := StrSplit(home_sections[1][2], "&")
+            ; home_sections.3/*[Subsection_ID]*/ := StrSplit(home_sections.1[3], "&")
+            
+
+            Total_Home_Sections := this.get_existing_sum_of_sections(home_sections)
+
+            dot_operator_start := 1
+
+            
+
+            loop % Total_Home_Sections
+            {
+                home_sections[dot_operator_start] := StrSplit(home_sections[0][dot_operator_start], "&")
+                dot_operator_start++
+            }
+            
+            ;msgbox % home_sections[2][2]
+
+            
+            
+
+
+            
+
+            ; msgbox % home_sections.2[1]
+            ; msgbox % home_sections.2[2]
+            ; msgbox % home_sections.2[3]
+
+            ; msgbox % home_sections.3[1]
+            ; msgbox % home_sections.3[2]
+            ; msgbox % home_sections.3[3]
+        ;*/
+        ;}
+
+        
+        return home_sections
+
+    }
+
+    Add_Section_Homes_Into_HomeStorage(home_sections, Minecraft_Window_Title)
+    {
+        result := this.get_existing_sum_of_sections(home_sections)
+
+        ; now that we got the amount of sections inside, all thats needed, is noting down the minecraft version. 
+        ; Since config is not a useful home section, subtract result by 1.
+
+
+
+        Insert_Home_numbers_into_HomeStorage(result, Minecraft_Window_Title)
+
+        TrayTip Minecraft Quick Homes, Added Homes&%result%&%Minecraft_Window_Title%
+
+        return
+    }
+
+    Get_Active_Window_Title()
+    {
+        WinGetActiveTitle, active_window_title
+
+        return active_window_title
+    }
+
+    ; Just pipe in the Parsed HomeStorage Array here.
+    get_existing_sum_of_sections(home_sections)
+    {
+        ;While home_sections has an index, execute inside code. 
+        while home_sections[0][A_Index]
+        {
+            ;msgbox % home_sections.1[A_Index]","A_Index
+            result := A_Index
+        }
+        
+
+        ;msgbox final index is %result%
+        ;If there is no homes index, then the result should be 0. 
+
+        return result
+    }
+
+}
+
+
+class switch_minecraft_header_sections
+{
+
+    current_header_section := 0
+
+    ; If no minecraft version is specified, aka `,0` then it will use Switch_Section_Hotkey
+    Switch_Set_Of_Homes_By_Sections(Switch_Section_Hotkey, Minecraft_Version)
```

Comment:
switch_minecraft_header_sections.Switch_Set_Of_Homes_By_Sections, can be changed to:
switch_minecraft_header_section.Switch_Section_by_catagory(Switch_Section_Hotkey, Minecraft_Version)

Reply:
Fixed in 158b07c9d6fe8962785b705e31eab84454b96102